### PR TITLE
VIH-6769 Updates_to_theres_a_problem_with_your_connection_error_page

### PR DESCRIPTION
**Pull request by RocheL**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-6769
### Change description ###
  - added call to profileService to check internet connection
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/services/error.service.spec.ts [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
 import { CallError } from '../waiting-space/models/video-call-models';
+import { ProfileService } from './api/profile.service';
 import { ErrorService } from './error.service';
 import { Logger } from './logging/logger-base';
 import { SessionStorage } from './session-storage';
 import { ErrorMessage } from '../shared/models/error-message';
 
 describe('ErrorService', () => {
     let router: Router;
+    let adalServiceSpy: jasmine.SpyObj<AdalService>;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/services/error.service.spec.ts:17]: could you avoid writing this with the testbed. this really hits the test run and even slower in the pipeline

Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/services/error.service.spec.ts]: updated to use the mockclasses

File: VideoWeb/VideoWeb/ClientApp/src/app/services/error.service.ts [hunk 2]
```diff
@@ -44,8 +53,16 @@ export class ErrorService {
         }
     }
 
+    private async checkIntenetConnection(): Promise<void> {
+        try {
+            await this.profileService.getProfileByUsername(this.adalService.userInfo.userName);
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/services/error.service.ts:58]: you can remove the need for AdalService by calling `getUserProfile` in the profile service, this does the same thing without relying on the username being manually provided.

Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/services/error.service.ts]: removed the adal service and changed call to getUserProfile

**Review by shaed-parkar — APPROVED** [review]
Looks good. Can you deploy this to one of the test environments before merging in please?

**Pull request closed without merging**